packages/docs/components/ColorCards/availableColors.ts

```diff
@@ -0,0 +1,105 @@
+import { ThemeInterface } from '@bigcommerce/big-design-theme';
+
+type Colors = ThemeInterface['colors'];
+type Color = keyof Colors;
+
+export interface ColorDetails {
+  contrast?: 'AAA' | 'AA' | 'A';
+  description?: {
+    text: string;
+    href?: string;
+  };
+  name: Color;
+  textColor: Color;
+}
+
+export const accentColors: ColorDetails[] = [
+  { contrast: 'AA', name: 'primary40', textColor: 'white' },
+  { contrast: 'AAA', name: 'primary10', textColor: 'secondary70' },
+  { contrast: 'AAA', name: 'primary20', textColor: 'secondary70' },
+  { contrast: 'AAA', name: 'primary30', textColor: 'secondary70' },
+  { contrast: 'AA', name: 'primary50', textColor: 'white' },
+  { contrast: 'AAA', name: 'primary60', textColor: 'white' },
+  { contrast: 'AAA', name: 'primary70', textColor: 'white' },
+];
+
+export const typeColors: ColorDetails[] = [
+  { contrast: 'AAA', name: 'secondary70', textColor: 'white' },
+  { contrast: 'AA', name: 'secondary60', textColor: 'white' },
+  { contrast: 'AA', name: 'secondary50', textColor: 'white' },
```

Comment:
The contrast levels should have the same color, right?  🤔  Noticed that `secondary50` have the contrast level as: A with color white and the other A in color black. Should they(AA) be with the same color, like the other ones?
 
 Example:
How it's right now:
<img width="156" alt="Screen Shot 2022-05-02 at 1 08 06 PM" src="https://user-images.githubusercontent.com/39739180/166301253-506c0292-07a4-4f3a-b012-1fb18df0c3e9.png">

How it think  it should be?:
<img width="271" alt="Screen Shot 2022-05-02 at 1 08 41 PM" src="https://user-images.githubusercontent.com/39739180/166301326-f8be0592-594f-46d4-b264-24757dfa6081.png">